todo_project/db/config.py

```
@@ -39,3 +39,7 @@ def check_database_health(self):
         except ConnectionFailure as e:
             logger.error(f"Failed to establish database connection: {e}")
             return False
+
+    def reset(self):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)